src/data/resolver/user.resolver.ts

```diff
@@ -0,0 +1,23 @@
+import { Arg, Mutation, Query, Resolver } from 'type-graphql';
+import { Repository } from 'typeorm';
+import { InjectRepository } from 'typeorm-typedi-extensions';
+import { UserEntity } from '../entity/user.entity';
+import { UserInputType } from '../type/user-input.type';
+import { UserOutputType } from '../type/user-output.type';
+
+@Resolver((of) => UserEntity)
+export class UserResolver {
+  constructor(@InjectRepository(UserEntity) private readonly userRepository: Repository<UserEntity>) {}
+
+  @Query((returns) => [UserOutputType])
```

Comment:
```suggestion
  @Query(() => [UserOutputType])
```

Quando o parâmetro não é usado, dá pra omitir.